### PR TITLE
Added bless spell to chaplain

### PR DESCRIPTION
**Pull request by Ahmed1Rashed**
:cl:
rscadd: Chaplain can bless items and people
/:cl:
The bless spell is purely cosmetic and doesnt actually change anything.
If the chaplain blesses an item, it has a Blessed prefix added to its name.

This is my first time tinkering with BYOND code. Any feedback is appreciated :smile:

### CODE REVIEW

File: code/modules/spells/spell_types/godhand.dm [hunk 1]
```diff
@@ -73,4 +73,34 @@
...
+
+/obj/item/weapon/melee/touch_attack/bless/afterattack(atom/target, mob/living/carbon/user, proximity)
+	if(!proximity)
+		return
+	if(user.lying || user.handcuffed)
+		return
+	if(isitem(target))
+		var/obj/effect/overlay/temp/heal/B = PoolOrNew(/obj/effect/overlay/temp/heal, get_turf(target))
+		B.color = "FFFF00"
+		user.visible_message("<font color = #FFBF00>[user] blesses the [target]!</font>")
```
Comment (Alek2ander) [inline code/modules/spells/spell_types/godhand.dm]: Span classes. Also, get rid of the 'the' unless you want 'the the banana peel'

File: code/modules/spells/spell_types/godhand.dm [hunk 2]
```diff
@@ -73,4 +73,34 @@
...
+		if(!cmptext("Blessed",copytext(target.name,1,8)))
+			target.name = "Blessed [target.name]"
```
Comment (Alek2ander) [inline code/modules/spells/spell_types/godhand.dm]: Lowercase please

File: code/modules/spells/spell_types/godhand.dm [hunk 3]
```diff
@@ -73,4 +73,34 @@
...
+	item_state = "bless"
+
+/obj/item/weapon/melee/touch_attack/bless/afterattack(atom/target, mob/living/carbon/user, proximity)
+	if(!proximity)
+		return
+	if(user.lying || user.handcuffed)
+		return
+	if(isitem(target))
+		var/obj/effect/overlay/temp/heal/B = PoolOrNew(/obj/effect/overlay/temp/heal, get_turf(target))
+		B.color = "FFFF00"
```
Comment (Alek2ander) [inline code/modules/spells/spell_types/godhand.dm]: ~~Blessed~~
Covered in piss

File: code/modules/spells/spell_types/godhand.dm [hunk 4]
```diff
@@ -73,4 +73,34 @@
...
+		if(!cmptext("Blessed",copytext(target.name,1,8)))
+			target.name = "Blessed [target.name]"
+			..()
+			return
+		..()
+		return
+	if(iscarbon(target))
+		var/obj/effect/overlay/temp/heal/B = PoolOrNew(/obj/effect/overlay/temp/heal, get_turf(target))
+		B.color = "FFFF00"
+		user.visible_message("<font color = #FFBF00>[user] blesses [target]!</font>")
```
Comment (Alek2ander) [inline code/modules/spells/spell_types/godhand.dm]: See above about spans and color

**Comment by Ahmed1Rashed** [comment]
Changed the things Alek2ander pointed out.

**Comment by Alek2ander** [comment]
You're still coloring everything in bright piss-yellow

**Comment by Kmc2000** [comment]
Yes

Yes

Yess

Make sure there is no limit to the amount of prefixes

Adam hits ian with the blessed blessed blessed blessed blessed blessed mint of corgi slaying (+5)

**Comment by Ahmed1Rashed** [comment]
screenshot 52

**Comment by Kmc2000** [comment]
M e r g e

**Comment by Cruix** [comment]
Re-add the prefix limit, don't listen to KMC

**Comment by Super3222** [comment]
:+1:

**Pull request merged**